data_model/src/isi.rs

```diff
@@ -342,11 +342,11 @@ isi! {
     #[display(fmt = "EXECUTE `{trigger_id}`")]
     #[serde(transparent)]
     #[repr(transparent)]
-    // SAFETY: `ExecuteTriggerBox` has no trap representation in `Trigger<FilterBox> as Identifiable>::Id`
+    // SAFETY: `ExecuteTriggerBox` has no trap representation in `Trigger<FilterBox, Executable> as Identifiable>::Id`
```

Comment:
I thought it is a good practice to put **"Safety"** section to doccomments ([reference](https://cheats.rs/#idiomatic-rust)). So, it will be also possible to leverage doc links.

Reply:
there is a difference between declaring `unsafe` function/trait and implementing one or writing an `unsafe` block of code.

Reply:
@0x009922 , can we close this?